### ISSUE TITLE
2 wlan

### ISSUE THREAD

**Issue by golku1** [v1s1t0r1sh3r3/airgeddon#334]
What is your airgeddon version?
v10.0

What is your Linux O.S. and which version?
Raspbian Buster

Which is the chipset of your wireless card?
AR9271

Describe the issue and the steps to reproduce it

Hi i am using airgeddon on my RPI
i am using wlan0 (intern wifi of RPI for wifi connection for vnc)
and using the wlan1 (USB) for monitor mode
after i have deauthed someone and got the handshake i am choosing the language and after that i get this message
The interface changed its name while setting in managed mode. Autoselected
after this message my connection got disconnected with the vnc because wlan0 is now used by airgeddon
is it possible to use wlan0 only for managed mode

**Comment by OscarAkaElvis** [comment]
Your problem is unclear... you must select one interface and only one interface is used all the time excepting for the DoS pursuit Mode. If your card is not supporting monitor mode don't select it and don't select DoS pursuit Mode. In that way you'll be using all the time the same interface which if is really working fine and supporting well monitor mode you'll have no problem.

You said you selected language... ¿? maybe are you talking about launguage for captive portal attack? please, be more specific, put screenshots. Otherwise is hard to understand what is exactly your problem.

**Comment by golku1** [comment]
oh sorry that i was not clear enough

when i connect the rpi with ethernet cable or just a monitor to see what happends everything is working without any problem

the problem is like this when i select the language 
i get this message
The interface changed its name while setting in managed mode. Autoselected
after that my connection with my pc to rpi gets disconnected because wlan0 is going to be used for the fake wifi

basically what i want is that airgeddon will only use wlan1 and leave wlan0

**Comment by OscarAkaElvis** [comment]
I think i know what is your problem... I guess that you are connecting to rpi by wifi and your rpi is using wlan0 to connect to your LAN. Even if you select wlan1 to perform the attacks, to create an AP the network manager is killed and that affects also to the wlan0. Please connect to your rpi by ethernet cable disabling your wlan0 wifi configuration and you'll work flawlessly.

I know you said "when i connect the rpi with ethernet cable..." but that is not true my friend... if the network manager is affecting you making lose connection to rpi is because you are connected to it by wifi on wlan0. Check carefully your configuration! good luck

**Comment by golku1** [comment]
that is true. never thinked about it thanks !
is it possible to remove wlan0 from the list of airgeddon? or do i need that from a other place ? because the fake ap is made with wlan0 what is really weak! i want to also do the fake ap with wlan1 that one is much stronger

**Comment by OscarAkaElvis** [comment]
you can't remove interfaces... but i think you are not understanding what i'm trying to say... no matter how many interfaces you have... to create a fake ap a "airmon-ng check kill" command is launched. It will affect to all your interfaces, no matter if listed or not in airgeddon... so what you need is:

1. connect to rpi without using any wlan interface. Ethernet cable is ok
2. select desired interface and launch the attack

I think is pretty clear. Closing.
